ui_testing/testcases/basic_tests/test006_orders.py

```diff
@@ -3657,4 +3658,22 @@ def test104_edit_order_page_then_overview(self, edit_case):
         else:
             self.assertFalse(self.order_page.confirm_popup(check_only=True))
             self.info('asserting redirection to active table')
-            self.assertEqual(self.order_page.orders_url, self.base_selenium.get_url())
\ No newline at end of file
+            self.assertEqual(self.order_page.orders_url, self.base_selenium.get_url())
+
+    def test107_analysis_number_displayed_in_order_child_table(self) :
+        """
+          Orders: Analysis number Approach: Make sure that the analysis number
+          should display in the order child table
+          LIMS-2622
+        """
+        self.orders_page.set_all_configure_table_columns_to_specific_value(
+            child=True, element='general:configure_child_table_items')
+        first_child_data = self.orders_page.get_child_table_data()
+        self.info("Check if Analysis No. displayed in Child Table headers")
+        self.assertIn('Analysis No.', first_child_data[0].keys())
+        self.info("Searching by the analysis number displayed in child table")
+        self.orders_page.filter_by_analysis_number(filter_text=first_child_data[0]['Analysis No.'])
+        import ipdb;ipdb.set_trace()
+        xpath = "//mark[contains(text(),'{}')]".format(first_child_data[0]['Analysis No.'].replace("'", ""))
```

Comment:
@0xIslamTaha I need you to tell me Is this check accepted to assert that analysis number highlighted by yellow color.
I tried to find any attribute called background-color or style, but no attributes shows style attached to this element